docs-conceptual/azps-13.2.0/mfa-requirement.md

```diff
@@ -0,0 +1,88 @@
+---
+description: Learn about the impact of mandatory multifactor authentication enforcement on Azure PowerShell in automation scenarios
+ms.custom: devx-track-azurepowershell
+ms.devlang: powershell
+ms.service: azure-powershell
+ms.topic: conceptual
+title: The impact of multifactor authentication on Azure PowerShell in automation scenarios
+---
+
+# The impact of multifactor authentication on Azure PowerShell in automation scenarios
+
+This article explores how multifactor authentication (MFA) affects automation tasks that use
+Microsoft Entra user identities and provides guidance on alternative approaches for uninterrupted
+automation.
+
+> [!IMPORTANT]
+> Action is required if you're using Microsoft Entra user identities for automation.
+
+MFA requirements prevent you from using Microsoft Entra user identities for authentication in
+automation scenarios. Organizations must switch to authentication methods designed for automation,
+such as managed identities or service principals, which support non-interactive automation use
+cases.
+
+## Limitations of user identities with MFA in automation
```

Comment:
I think it would be helpful to be more precise about the flow that will fail and maybe give a sample of the command + error message so that it can be discoverable ?

Reply:
A year or so ago I tested, probably in PowerShell (?), and received an "interactive authentication is needed" message which I documented in authenticate-azure-cli-interactevely.md. Today in PowerShell I first get an error "Unable to load extension..." but just as quickly a pop-up window to authenticate. Then, interestingly, a BAMI "Authentication failed against tenant..." error, followed by a subscription picker of MSFT tenant subscriptions. I'll put a screen print in our Team's chat.

For an automated process, the application should get the "interactive authentication is needed" error.

Reply:
I added a note to the article with the error message: "Interactive authentication is needed". If additional error messages are discovered, we can add a dedicated section for them.